### PR TITLE
Migration of remaining unit test files

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

Migration of remaining unit test files

https://github.com/medic/angular10-migration/issues/126

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: webapp/angular.json [hunk 1]
```diff
@@ -90,7 +90,7 @@
         "test": {
           "builder": "@angular-devkit/build-angular:karma",
           "options": {
-            "main": "tests/karma/ts/test.ts",
+            "main": "tests/karma/test.ts",
```
Comment (latin-panda) [inline webapp/angular.json:93]: The `test.ts` in this level will recognise specs from `/js` and `/ts` directories

File: webapp/src/ts/reducers/contacts.ts [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+import { Actions } from '@mm-actions/contacts';
 
 const initialState = {
   contacts: [],
   contactsById: new Map(),
   selected: null,
   filters: {},
   loadingSelectedChildren: false,
-  loadingSelectedContacts: false,
+  loadingSelectedReports: false,
```
Comment (latin-panda) [inline webapp/src/ts/reducers/contacts.ts:13]: `loadingSelectedContacts` isn't used anymore in the code.
Added `loadingSelectedReports` in state definition.

**Comment by latin-panda** [comment]
Hi @njogz Can you please take a look at this PR? 
I've added a little modification to the contact's reducer, can check if that's okay? Thank you!

**Review by njogz — APPROVED** [review]
Good job!

**Pull request merged**